ansible/roles/common/tasks/user.yml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I think we should probably change these scripts so that they use the home directory env var in the context they are executed and not add this here?

Reply:
if they accepts relative paths we can use `~/` instead of full one yes, I could try it in the grc (I don't use the twitter part) if it's called on a standard pass

Reply:
Or we can use one of the Python ways of getting the home dir e.g. `os.path.expanduser("~")`. I think I would prefer trying to do that in this PR rather than this trick :see_no_evil:

Reply:
done for the python script, I'm testing with `~/` in the grc and will see on the next meteor pass if I see any error in the logs

Reply:
I haven't seen errors in logs, but I'm not really sure how this grc is called and when, my METEOR passes (gnuradio or rtl_fm) never generate anything here, probably too weak signal or something.